roles/openshift_logging/templates/es.j2

```diff
@@ -95,6 +95,13 @@ spec:
               readOnly: true
             - name: elasticsearch-storage
               mountPath: /elasticsearch/persistent
+          readinessProbe:
+            exec:
+              command:
+              - "/usr/share/elasticsearch/probe/readiness.sh"
+            initialDelaySeconds: 5
+            timeoutSeconds: 5
```

Comment:
@lukas-vlcek 10s is the default for PeriodSeconds.
But those are only comments... the actual defaulting happens [here](https://github.com/openshift/origin/blob/e37455a407f05ac584f5c462effc2b8c3626f39d/vendor/k8s.io/kubernetes/pkg/api/v1/defaults.go#L184-L197).